src/components/Reports/GoalCalculator/SharedComponents/GoalCalculatorGrid/GoalCalculatorGrid.tsx

```diff
@@ -278,7 +270,7 @@ const GoalCalculatorGridForm: React.FC<GoalCalculatorGridFormProps> = ({
           </Box>
         ) : (
           <>
-            <StyledControlsBox>
+            <Box>
```

Comment:
I believe this makes an empty div. Can you remove the `Box` unless it's needed for something?